### PR TITLE
Notifications Next Level

### PR DESCRIPTION
**Pull request by basticks02**
In this PR, I will be implementing the live notifications based on appointment creation - from the physicians end to the patients end. 

I've set up the database and implemented  the post and get of notifications to the notifications page. Check the video below:

https://www.loom.com/share/c29bd4030b2f4c3fbc43394213e365db?sid=65d0de95-26a1-433d-abd8-f18f0f51e0f3

### CODE REVIEW

File: backend/user.js [hunk 1]
```diff
@@ -254,12 +254,21 @@ router.post('/appointments', authenticateToken, async (req, res) => {
...
-    }
+
+    const newNotification = await prisma.notification.create({
+      data: {
+        content: `New appointment created: ${title} on ${date}`,
+        patientId: patient.id,
+      },
+    });
+
+    // console.log(userToWS[patientId])
```
Comment (m5matthew) [inline backend/user.js]: Can we remove comments?

**Review by m5matthew — APPROVED** [review]
Nice, this doesn't include the websockets yet right? Just setting up the backend & showing notifs on refresh?

**Comment by basticks02** [comment]
https://share.icloud.com/photos/0b3Dsz5cU9-9pCA0xC7KFBEow

(The screen recording size was too large)

File: backend/user.js (hunk 1 above)
Comment (basticks02) [reply backend/user.js]: Oh yes. Sorry about that.

**Pull request merged**